agent-install/agent-install.sh

```diff
@@ -18,8 +18,11 @@ SKIP_REGISTRATION=false
 CFG="agent-install.cfg"
 OVERWRITE=false
 HZN_NODE_POLICY=""
+AGENT_INSTALL_ZIP="agent-install-files.tar.gz"
+NODE_ID_MAPPING_FILE="node-id-mapping.csv"
 
 VERBOSITY=3 # Default logging verbosity
+BATCH_INSTALL=false
 
 # required parameters and their defaults
 REQUIRED_PARAMS=( "CERTIFICATE" "HZN_EXCHANGE_URL" "HZN_FSS_CSSURL" "HZN_ORG_ID" "HZN_EXCHANGE_USER_AUTH" )
```

Comment:
I removed the part that adds the cert to the device's trust store. Made the certificate optional, when it is there it will get set in /etc/default/horizon.